README.md

```diff
@@ -88,9 +88,9 @@ Please see [Contributing to WebAssembly](https://github.com/WebAssembly/design/b
 [multi-memory]: https://github.com/WebAssembly/multi-memory
 [tail_call]: https://github.com/WebAssembly/tail-call
 [threads]: https://github.com/webassembly/threads
-[type_reflection_for_webassembly_javascript_api]: https://github.com/WebAssembly/js-types
+[js-types]: https://github.com/WebAssembly/js-types
 [wasm_c_api]: https://github.com/WebAssembly/wasm-c-api
-[web_content_security_policy]: https://github.com/WebAssembly/content-security-policy
+[content-security-policy]: https://github.com/WebAssembly/content-security-policy
```

Comment:
It looks like the convention here is to use underscores rather than dashes in the names.

Reply:
I think it makes sense to use the name of the repo.. even though I've not changed all them here.